rootfs/etc/nginx/lua/monitor.lua

```diff
@@ -91,7 +105,10 @@ function _M.call()
   end
 
   metrics_count = metrics_count + 1
-  metrics_batch[metrics_count] = metrics()
+  local metrics_str = metrics
+  if metrics_str then
+    metrics_batch[metrics_count] = metrics_str
```

Comment:
The original code is a Lua table, and it is a string type after modification.
Is this wrong?

Reply:
no wrong. because encoding consumes expensive CPU resources, and the encoding process is evenly distributed to each request, so when the metrics of each request are generated, the encoding is already done.